src/lib/beta-features.ts

```diff
@@ -14,6 +14,12 @@ const BETA_FEATURES_DEFINITION: Record< keyof BetaFeatures, BetaFeatureDefinitio
 		default: false,
 		description: '"studio site" command to manage local sites from terminal',
 	},
+	createSiteFromRemote: {
+		label: 'Create Site from Remote',
```

Comment:
Would it be clearer?

```suggestion
		label: 'Create a site from existing remote site',
```